brightcove/brightcove/src/page/Home.tsx

```diff
@@ -52,14 +52,14 @@ const Home: React.FC = () => {
       if (items && typeof items[0] !== 'object') {
         const idArray =
           items.length > 10 ? splitItemsArray(items) : [items.toString()];
-
-        const brightcove = new Brightcove(extensions.config.proxyUrl);
+        const { proxyUrl, oauthUrl, brightcoveUrl } = extensions.config;
```

Comment:
Better way to write this -->

`const { config: { proxyUrl, oauthUrl, brightcoveUrl } = {} } = extensions;`